### PR TITLE
WPE* Version bumps

### PR DESCRIPTION
**Pull request by philn**

### CODE REVIEW

File: recipes-browser/wpebackend-fdo/wpebackend-fdo.inc [hunk 1]
```diff
@@ -9,5 +9,3 @@ PROVIDES += "virtual/wpebackend"
 RPROVIDES_${PN} += "virtual/wpebackend"
 
 inherit cmake
-
-FILES_${PN} += "${libdir}/libWPEBackend-fdo-0.1.so"
```
Comment (psaavedra) [inline recipes-browser/wpebackend-fdo/wpebackend-fdo.inc]: Shouldn't this be moved to  the wpebackend-fdo_0.1.bb recipe

Comment (philn) [reply recipes-browser/wpebackend-fdo/wpebackend-fdo.inc]: It's removed. Not sure what you mean :)

**Comment by philn** [comment]
Ah, forgot to rename wpebackend to libwpe. Hold on :)

File: recipes-browser/wpebackend-fdo/wpebackend-fdo.inc (hunk 1 above)
Comment (philn) [reply recipes-browser/wpebackend-fdo/wpebackend-fdo.inc]: I have no idea why, the symlink is not included in the package containing the libraries (`so` files) but instead is part of the wpebackend-fdo-dev package... I'll leave resolving this issue to an actual Yocto expert and include the -dev pkg in my image meanwhile.

Comment (philn) [reply recipes-browser/wpebackend-fdo/wpebackend-fdo.inc]: I have no idea why, the symlink is not included in the package containing the libraries (so files) but instead is part of the wpebackend-fdo-dev package... I'll leave resolving this issue to an actual Yocto expert and include the -dev pkg in my image meanwhile.

**Comment by philn** [comment]
Ping @psaavedra :)

File: recipes-browser/wpewebkit/wpewebkit_2.21.91.bb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+SRC_URI = "https://wpewebkit.org/releases/${PN}-${PV}.tar.xz \
+           file://0001-REGRESSION-r217927-WPE-2.22-GSTREAMER_GL-Video-appea.patch"
+
+SRC_URI[md5sum] = "08778a95eba2de141d8aaff3d802c811"
+SRC_URI[sha256sum] = "e75b1ca225fa9303017e954b6203e4f1750a7566fa852443e6fd574fd5f926b8"
+
+DEPENDS += " libwpe"
+RCONFLICTS_${PN} = "wpebackend (< 1.0) wpebackend-fdo (< 1.0)"
```
Comment (macpijan) [inline recipes-browser/wpewebkit/wpewebkit_2.21.91.bb]: @philn I am getting:
`Problem: package wpewebkit-2.21.91-r0.armv7vet2hf_neon conflicts with wpebackend < 1.0 provided by libwpebackend-0.1-0-0.1-r0.armv7vehf_neon` Is there a recipe fot the `wpebackend` 1.0? I can only see `0.1` version.

Comment (philn) [reply recipes-browser/wpewebkit/wpewebkit_2.21.91.bb]: There is a recipes-browser/libwpe/libwpe_1.0.0.bb ... WPEBackend was renamed to libwpe just before the 1.0 release.

Comment (macpijan) [reply recipes-browser/wpewebkit/wpewebkit_2.21.91.bb]: Or maybe the idea is that the wpebackend should not be installed at all and the libwpe should be used instead

Comment (macpijan) [reply recipes-browser/wpewebkit/wpewebkit_2.21.91.bb]: Yeah, just figured that out. Thanks

**Comment by jnettlet** [comment]
Additionally this patchset has an issue with the cog versioning in relation to wpewebkit.  If you build wpewebkit 2.20.2 then cog needs a SRCREV of 4487c4ce53f5489c244655d9ea0feea90034ab82.
It is probably best if we set a flag to dictate if the build should be against stable revisions, or devel revisions.

**Comment by philn** [comment]
Hum yes, perhaps we can have a specific recipe for the cog 0.1 branch?

File: recipes-browser/wpebackend-fdo/wpebackend-fdo.inc (hunk 1 above)
Comment (psaavedra) [reply recipes-browser/wpebackend-fdo/wpebackend-fdo.inc]: working in rocko at least, just with this change:

```
+FILES_SOLIBSDEV = ""
+FILES_${PN} += "${libdir}/libWPEBackend-default.so ${libdir}/libWPEBackend-fdo-0.1.so"
+INSANE_SKIP_${PN} ="dev-so"
+
+do_install_append () {
+    install -d "${D}/usr/lib"
+    ln -s libWPEBackend-fdo-0.1.so "${D}/usr/lib/libWPEBackend-default.so"
+}
```


`FILES_SOLIBSDEV=""` forces all libraries to be included in the RPM package

Comment (psaavedra) [reply recipes-browser/wpebackend-fdo/wpebackend-fdo.inc]: Sent a PR for this https://github.com/Igalia/meta-webkit/pull/37

**Pull request merged**